src/components/EmployeeTable/EmployeeTableRow.jsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faEye, faEdit, faTrash } from "@fortawesome/free-solid-svg-icons";
+
+import styles from "../../styles/users.module.css";
```

Comment:
@Doyin02 move the styles you need to this component or it's parent css.module file, then reference them. 

If it's the parent, then it would be:
```suggestion
import styles from "./EmployeeTable.module.css";
```